src/App.css

```diff
@@ -35,3 +34,57 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+/****** HOME PAGE STYLES ******/
```

Comment:
If these are only styles for the home page, then we should probably put them in their own css file. Or better yet use React Inline styles. 

https://zhenyong.github.io/react/tips/inline-styles.html

We should only use App.css for things for our global for the entire application. 

http://www.material-ui.com/#/customization/styles

Reply:
I'll put them in their own css file, unless you think inline is a better option.

Reply:
For css file naming convention, should it be <page-name>.css or style.css ?